src/router/AdminRoute.tsx

```diff
@@ -1,13 +1,13 @@
 import React from "react";
-import { isAdmin } from "../utils/mocksettings.json";  //mocked admin
+// import { isAdmin } from "../utils/mocksettings.json";  //mocked admin
 import { Route, Redirect } from "react-router-dom";
 interface props {component: Function, path:string}
 
 const AdminRoute = ({component: Component, ...rest}:props) => {
     return(
         <Route {...rest}
                render={props => {
-                   if(isAdmin) return <Component {...props} />;
+                   if(localStorage.getItem('admin') === "true") return <Component {...props} />;
```

Comment:
is not necessary to do `=== "true"` comparison. Only this `if(localStorage.getItem('admin'))` should work